src/ni/vsbuild/steps/LvUnitTestFrameworkStep.groovy

```diff
@@ -0,0 +1,17 @@
+package ni.vsbuild.steps
+
+import ni.vsbuild.BuildConfiguration
+
+class LvUnitTestFrameworkStep extends LvProjectStep {
+
+   def UTF_path
+
+   LvUnitTestFrameworkStep(script, mapStep, lvVersion) {
+      super(script, mapStep, lvVersion)
+      this.UTF_path = mapStep.get('UTF_path')
```

Comment:
The variable in TOML should be `utf_path` to match the existing conventions.

Reply:
Removed UTF